enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/catchup/storecopy/StoreCopyRequestHandler.java

```diff
@@ -63,32 +66,38 @@ public GetStoreFileRequestHandler( CatchupServerProtocol protocol, Supplier<NeoS
         this.storeFileStreamingProtocol = storeFileStreamingProtocol;
         this.pageCache = pageCache;
         this.fs = fs;
-        this.log = logProvider.getLog( GetStoreFileRequestHandler.class );
+        this.log = logProvider.getLog( StoreCopyRequestHandler.class );
     }
 
     @Override
-    protected void channelRead0( ChannelHandlerContext ctx, GetStoreFileRequest fileRequest ) throws Exception
+    protected void channelRead0( ChannelHandlerContext ctx, T request ) throws Exception
     {
-        log.debug( "Requesting file %s", fileRequest.file() );
+        log.debug( "Handling request " + request );
```

Comment:
This will create the message string every time, even if debug logging is not on. The parameterised method would avoid that.